src/hotspot/share/gc/shenandoah/shenandoahOldGC.cpp

```diff
@@ -129,21 +131,15 @@ bool ShenandoahOldGC::collect(GCCause::Cause cause) {
   // Complete marking under STW
   vmop_entry_final_mark();
 
-  entry_old_evacuations();
-
   // We aren't dealing with old generation evacuation yet. Our heuristic
   // should not have built a cset in final mark.
   assert(!heap->is_evacuation_in_progress(), "Old gen evacuations are not supported");
 
-  // Concurrent stack processing
-  if (heap->is_evacuation_in_progress()) {
```

Comment:
Where has this gone?

Reply:
This was unreachable code (see assert just above on line 136). The old generation defers all evacuation to _mixed_ collections.